package.json

```diff
@@ -42,6 +42,7 @@
     "url": "git://github.com/ampersandjs/ampersand-form-view"
   },
   "scripts": {
+    "validate": "jshint .",
```

Comment:
Oh, gotcha. Does that mean changing from "validate" to "lint" would prevent it running twice? That way there's still a shorthand available.